.circleci/scripts/gke_ci_cluster/create_cluster.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#Exports.
+export GCP_PROJECT_NAME=dai-dev-55
+export GKE_CLUSTER_NAME=gke-circleci
+export GKE_REGION=us-west1
+export GKE_MACHINE_TYPE=n1-standard-8
+export GKE_NUM_NODES=1
+export GCS_NETWORK_NAME=gke-circleci-vpc
+
+export BASTION_INSTANCE_NAME=gke-circleci-bastion
+export BASTION_INSTANCE_ZONE="$GKE_REGION-b"
+export BASTION_INSTANCE_TYPE=e2-micro
```

Comment:
Ideally, we would want to run this manually to keep the cluster up indefinitely so that CI jobs can connect to and use it, so this deployment script itself wasn't meant to be executed by CI jobs. 
In that case, should I move this into our `tools` directory?

Reply:
Done, merged [there](https://github.com/determined-ai/dev-scripts/pull/37).